src/applications/representative-appoint/components/SearchResult.jsx

```diff
@@ -0,0 +1,174 @@
+    //   'search-results-id': representativeId,
+    //   'search-results-total-count':
+    //     searchResults?.meta?.pagination?.totalEntries,
+    //   'search-results-total-pages': searchResults?.meta?.pagination?.totalPages,
+    //   'search-result-position': key,
+    // });
+  };
+
+  return (
+    <va-card class="representative-result-card vads-u-padding--4">
+      <div className="representative-result-card-content">
+        <div className="representative-info-heading">
+          {distance && (
+            <div
+              id={`representative-${representativeId}`}
+              className="vads-u-font-weight--bold vads-u-font-family--serif"
+            >
+              {parseFloat(JSON.parse(distance).toFixed(2))} Mi
+            </div>
+          )}
+          {officer && (
+            <>
+              <div
+                className="vads-u-font-family--serif vads-u-margin-top--2p5"
+                id={`result-${representativeId}`}
+              >
+                <h3 aria-describedby={`representative-${representativeId}`}>
+                  {officer}
+                </h3>
+              </div>
+              {associatedOrgs?.length === 1 && (
+                <p style={{ marginTop: 0 }}>{associatedOrgs[0]}</p>
+              )}
+            </>
+          )}
+        </div>
+        {associatedOrgs?.length > 1 && (
+          <div className="associated-organizations-info vads-u-margin-top--1p5">
+            <va-additional-info
+              trigger="See associated organizations"
+              disable-border
+              uswds
+            >
+              {associatedOrgs?.map((org, index) => {
+                return (
+                  <>
+                    <p>{org}</p>
+                    {index < associatedOrgs.length - 1 ? (
+                      <br style={{ lineHeight: '0.625rem' }} />
+                    ) : null}
+                  </>
+                );
+              })}
+            </va-additional-info>
+          </div>
+        )}
+
+        <div className="representative-contact-section vads-u-margin-top--3">
+          {addressExists && (
+            <div className="address-link">
+              <a
+                href={`https://maps.google.com?saddr=${
+                  query?.context?.location
+                }&daddr=${address}`}
+                tabIndex="0"
+                className="address-anchor"
+                onClick={() => recordContactLinkClick()}
+                target="_blank"
+                rel="noreferrer"
+                aria-label={`${address} (opens in a new tab)`}
+              >
+                {addressLine1}{' '}
+                {addressLine2 ? (
+                  <>
+                    <br /> {addressLine2}
+                  </>
+                ) : null}{' '}
+                <br />
+                {city}, {stateCode} {zipCode}
+              </a>
+            </div>
+          )}
+          {phone && (
```

Comment:
Phone numbers aren't showing up for me locally when I can see the attribute exists in the network tab

Reply:
I can update those today. Good to know